src/task/MonitorTask.ts

```diff
@@ -262,6 +262,15 @@ export default class MonitorTask
   }
 
   connectionHealthDidChange(connectionHealthData: ConnectionHealthData): void {
+    if (!this.context.meetingSessionConfiguration?.urls?.audioHostURL?.length) {
```

Comment:
Isn't a ping-pong check based on a WebSocket connection only? Or does the media server require an audio connection to start a ping-pong check?

Reply:
line was reverted.

Reply:
Thanks for checking